### ISSUE TITLE
[Bug] `PHPunit bugs and best practices` false positive assertion suggestion

### ISSUE THREAD

**Issue by vladyslavstartsev** [kalessil/phpinspectionsea#1152]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Issue type     | Bug/False-Positive, Feature request, Support Request                          |
| Plugin         | Php Inspections (EA Extended), v3.0.9.1 |
| Language level | PHP 7.2     |

Current behaviour (description/screenshot:)
--
this
```php
$this->assertTrue(is_array($this->manager->driver('sqlite')));
```
suggest to be converted to this
```php
$this->assertInternalType('array', $this->manager->driver('sqlite'));
```
Expected behaviour
--
It should not be suggested to do that because since phpunit 7.4 (not sure about the version) `assertInternalType` is deprecated

see  https://github.com/sebastianbergmann/phpunit/issues/3369
also see as an example https://github.com/laravel/framework/pull/26976 for changes made

**Comment by fruitl00p** [comment]
@kalessil this might be related to #1155

**Comment by kalessil** [comment]
Fixed!
